system_test.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+""" This is the system test for texasbbq.
+
+It tests that the latest released version of Numba with the latest released
+version of umap. If anything is red, it's probably due to a bug with texasbbq.
```

Comment:
I presume `umap` is chosen for it's stability and use of just Numba's jit decorators?

Reply:
Indeed, it's most likely to always be successful.